packages/retail-ui/components/Tooltip/__tests__/Tooltip-test.tsx

```diff
@@ -115,30 +97,27 @@ describe('Tooltip', () => {
     expect(wrapper.find(StatefulComponent).length).toBe(1);
   });
 
-  it('reset opened state by `tigger="closed"` prop', async () => {
+  it('reset opened state by `tigger="closed"` prop', () => {
     const Content = () => <div />;
 
     const wrapper = mount<TooltipProps>(
```

Comment:
Тут неправильный тип для дженерика проставлен.
```suggestion
     const wrapper = mount<Tooltip>(
```

Reply:
done